docs/how-to-guides/deployment-options.md

```diff
@@ -89,6 +89,11 @@ The table below outlines the parameters that can be configured through any of th
 | securityEnabled | boolean | No | Enable or disable security settings for the APIs. | false |
 | securityAuthenticationAudiences | array of strings | No | The audiences for the API authentication. If securityEnabled is set to true, this parameter must have a value provided. | empty array |
 | securityAuthenticationAuthority | string | No | The authority for the API authentication. If securityEnabled is set to true, this parameter must have a value provided. | empty string |
+| storageAccountNetworkIsolationEnabled | boolean | No | This is an optional security enhancement. If set to true, the template Storage Account will only be accessible to resources within a specified Virtual Network (one will be created if it doesn't already exist), and the Container Apps environment will be deployed on the Virtual Network so that it can access the Storage Account. | false |
```

Comment:
nit: Similar to my other comment, VNet's can be configured with ACA Env even without the storage account necessitating it, so this param could be more generic like networkIsolationEnabled perhaps.

Reply:
But fine to leave as is since I see below the documentation is more specifically addressing the storage account n/w isolation.